internal/fabric_utils/correlation_id.go

```diff
@@ -0,0 +1,24 @@
+package fabric_utils
```

Comment:
the package name should avoid `utils`

Without looking at how this is used.. fabric/correlation ?

If this is only used in one or two files, especially for testing, I'd be tempted to make this a private helper adjacent to the code that needs it.

Reply:
There will be more fabric utils when we migrate further:

```
fabric_mapping_helper.go
fabric_mapping_service_profile_helper.go
fabric_port_mapping_helper.go
```

The CorrelationId code was in fabric_mapping_helper.go IIRC. I didn't think about it too much and moved it to fabric_utils. However it seems that CorrelationId is not used anywhere but in Config. So I moved it straight to Config.

Reply:
You can move that CorrelationId code into Config but I would create a `helper` folder under `internal`, you can add those mapping helpers you mention there and we will also need that also to avoid import cycles https://github.com/equinix/terraform-provider-equinix/pull/406#issuecomment-1764477274 so it should look like 
![image](https://github.com/equinix/terraform-provider-equinix/assets/68897552/8b1e1d97-f310-4630-9039-649e3c866d84)

Reply:
`helper` (like `utils`) is too generic of a name

- https://go.dev/doc/effective_go#names
- https://go.dev/blog/package-names#bad-package-names